vllm/v1/worker/gpu_model_runner.py

```diff
@@ -1042,6 +1043,11 @@ def execute_model(
                 for k, v in self.intermediate_tensors.items()
             })
 
+        # Update the connector's state with the metadata in scheduler output.
+        if has_kv_transfer_group():
```

Comment:
I wonder if it makes sense to do the setting and clearing in in the `set_forward_context()`?

Reply:
QQ: Why not execute this at the beginning of `execute_model`? Could you please describe (in the code) at which point of time this API should be called?